### ISSUE TITLE
Wrong logo and colors for Google Sign In

### ISSUE THREAD

**Issue by crutchcorn** [sac-tech-org/job-board#7]
![image](https://github.com/sac-tech-org/job-board/assets/9100169/b97b1b96-b5cf-4b20-9ea5-8aaaf6eb3573)

Shouldn't be the GitHub logo and colors for the Google sign in

**Comment by rusher2004** [comment]
Yeah this was just direct copy/paste html from a sample by SuperTokens. A really nice solution for this would be to update the Button component to be able to take in an icon property to nest in the button like this.
